src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[3];
+  const toSeparator = toFormat[3];
+  const originalDate = date.split(fromSeparator);
+
+  const newDate = [];
```

Comment:
Delete an extra empty line in blocks with the same logic
```suggestion
  const originalDate = date.split(fromSeparator);
  const newDate = [];
```